osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -249,11 +262,13 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                     },
                     email=False
                 )
+                logger.exception()
                 raise
             finally:
                 try:
                     subprocess.check_call("docker-compose stop --timeout 0".split(' '))
                     subprocess.check_call("docker-compose rm -v -f".split(' '))
                 except:
+                    logger.exception("couldn't clean up docker containers")
```

Comment:
I think it's really important to see the container status at the moment it failed. I agree `docker-compose ps` is better than `docker ps -a`. I think we should try to get this information, catch every errors may occour and pass the information we got.